### ISSUE TITLE
Wrong version display on 8.7.21?

### ISSUE THREAD

**Issue by surak** [TACC/Lmod#642]
Hey Robert, maybe I'm doing something wrong, but the latest one still gives me the wrong version number:

```
Modules based on Lua: Version 8.7.20 (8.7.20-4-gafc025c2) 2023-03-16 11:43 -05:00
    by Robert McLay mclay@tacc.utexas.edu
```

**Comment by rtmclay** [comment]
Thanks for pointing this out. I just released 8.7.22.  The only change is getting the version display to be correct.

**Comment by surak** [comment]
Great! And here is the deb package: https://github.com/surak/Lmod/releases/tag/8.7.22
